src/app/components/ScrollablePromo/index.test.jsx

```diff
@@ -157,4 +166,16 @@ describe('ScrollablePromo', () => {
     'it should match a11y snapshot for list',
     <ScrollablePromoWithContext blocks={threeLinks} />,
   );
+
+  it('should render recommendation variation when recommendation && recommendation data are passed', () => {
```

Comment:
From the issue, is says `This will show a max of 3 promos in the component`. I would also include a test to make sure only 3 recs are shown when an array has more than 3 recs passed to it.

Reply:
I would also test this path `recommendationTitle`. While testing `You may also be interested in` as a fallback works, you should also have a test for `recommendationTitle` to make sure it shows the title that is in the data.

Reply:
> From the issue, is says `This will show a max of 3 promos in the component`. I would also include a test to make sure only 3 recs are shown when an array has more than 3 recs passed to it.

We already test this in previous tests for the normal EOJ, I don't believe repeating the same test with recommendation data is worth it. I see it as test repetition, but happy to add it if you think otherwise 👍

Reply:
That should be good; no need to add that case then.

Reply:
One thing I just noticed might be useful is to have these tests in a separate describe block (within `ScrollablePromo` describe) called `recommendationEOJ` just to clarify that these cases specifically to the rec edoj